### PR TITLE
feat: new decorator info-override

### PR DESCRIPTION
**Pull request by RomanHotsiy**
## What/Why/How?

This decorator will merge/override additional properties into the info object.

With this decorator you cannot remove some properties from info. Do we need another decorator or some option in this to switch from "merge" to "complete replace" behaviour?

Thoughs?

## Testing

Added tests.

## Check yourself

- [x] Code is linted
- [x] Tested with redoc/reference-docs/workflows
- [ ] All new/updated code is covered with tests

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

**Review by IgorKarpiuk — APPROVED** [review]

File: __tests__/bundle/info-override/.redocly.yaml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apis:
+  main:
+    root: ./main.yaml
+
+decorators:
+  info-override:
+    title: Updated title
+    x-vendor: custom extension
```
Comment (IgorKarpiuk) [inline __tests__/bundle/info-override/.redocly.yaml:8]: Should  there be a descriptions like in snapshot ?

 ```yaml
 description: |
    This is a test API description.

    # Heading test

    Body [test](#) content.
```

Comment (RomanHotsiy) [reply __tests__/bundle/info-override/.redocly.yaml:8]: You are right

Comment (adamaltman) [reply __tests__/bundle/info-override/.redocly.yaml:8]: From the docs it looks like it should not need it. It overrides properties and leaves the ones that were not in the override alone, right?

Comment (adamaltman) [reply __tests__/bundle/info-override/.redocly.yaml:8]: Nevermind, the original and new one have different descriptions. 🤣

Comment (RomanHotsiy) [reply __tests__/bundle/info-override/.redocly.yaml:8]: Should be resolved now.

**Pull request merged**